src/libinjector/injector_stack.c

```diff
@@ -481,7 +481,7 @@ bool setup_linux_stack(vmi_instance_t vmi, x86_registers_t* regs, struct argumen
                 regs->r8 = args[4].data_on_stack;
             // fall through
             case 4:
-                regs->rcx = args[3].data_on_stack;
+                regs->r10 = args[3].data_on_stack;
```

Comment:
are you using this function somewhere?

Reply:
yes, when I am doing `setup_stack` while setting up `mmap` call, it is being called internally

Reply:
( I had created a seperate PR to discuss this change also, you can have a look at it https://github.com/tklengyel/drakvuf/pull/1268 )

Reply:
Done